org.eclipse.wildwebdeveloper.xml/src/org/eclipse/wildwebdeveloper/xml/internal/ui/messages.properties

```diff
@@ -23,6 +23,16 @@ XMLCatalogPreferencePage_OpenInEditorApplyAndEdit=Apply and Edit
 # XML CodeLens preference page
 XMLCodelensPreferencePage_codelens_enabled=Enable XML &CodeLens
 
+# XML Formatting preference page
+XMLFormattingPreferencePage_format_emptyElements=Expand/collapse empty elements
+XMLFormattingPreferencePage_format_spaceBeforeEmptyCloseTag=Insert space before end of self closing tag
+XMLFormattingPreferencePage_format_splitAttributes=&Split multiple attributes each onto a new line
```

Comment:
Split is probably not the right term. I suggest `Place attributes on separate lines`

Reply:
```suggestion
XMLFormattingPreferencePage_format_splitAttributes=&Place attributes on separate lines
```

Reply:
I would like to keep this label since we use it too in vscode-xml and WTP uses too this label:

![image](https://user-images.githubusercontent.com/1932211/188462338-037b7264-fdc4-459b-873c-e170f2f2641c.png)